operators/constellation-node-operator/main.go

```diff
@@ -196,6 +219,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	wg := &sync.WaitGroup{}
+	exec.Start(context.Background(), wg)
+	defer wg.Wait()
```

Comment:
avoid deferring correct call order to caller. instead encapsulate inside start() (see suggestion)